### ISSUE TITLE
Feature Request: Dashboard Widgets

### ISSUE THREAD

**Issue by garrettloy** [jeffpriz/devops-pr-stats#13]
The entire PR stats hub is great! We would like to pick and choose sections to throw on our team dashboard that we look at together daily, in particular the PR Code Reviewers breakdown with a configurable lookback time, but almost all of the sections in the hub would look good as widgets in the dashboards section.

Thanks for your work on this!

**Comment by gymbratjodee** [comment]
Yes please!!

**Comment by jeffpriz** [comment]
yeah, I love the thought there.  I haven't built out a Dashboard widget, but that has been a request I've received a bit, would love to make it happen

**Comment by bmolloy** [comment]
I'd also love this.  Great extension, but for me the ability to see stats across all repos in a project would be the icing on the cake.
